packages/nx-plugin/src/utils/testing-utils/paths.ts

```diff
@@ -1,3 +1,7 @@
+export function tmpFolder() {
+  return `${process.cwd()}/tmp`;
```

Comment:
I know this is existing behavior, but we may want to reevaluate this in the future. `process.cwd() !== workspaceRoot` when running tests from inside a directory.

e.g.:

1. nx e2e e2e
2. {workspaceRoot}/tmp is created

vs

1.  cd e2e
4. nx e2e
5. {workspaceRoot}/e2e/tmp is created

Reply:
changed process.cwd() to workspaceRoot